docs/UsersGuide/source/BuildRunSRW.rst

```diff
@@ -601,7 +633,56 @@ For WCOSS_CRAY:
    ``valid_param_vals script``. In addition, various example configuration files can be
    found in the ``regional_workflow/tests/baseline_configs`` directory.
 
+.. _VXConfig:
+
+Configure METplus Verification Suite (Optional)
+--------------------------------------------------
+
+Users who want to use the METplus verification suite to evaluate their forecasts need to add additional information to their ``config.sh`` file. Other users may skip to the :ref:`next section <SetUpPythonEnv>`. 
+
+.. attention::
+   METplus *installation* is not included as part of the build process for this release of the SRW App. However, METplus is preinstalled on `Level 1 <https://github.com/ufs-community/ufs-srweather-app/wiki/Supported-Platforms-and-Compilers>`__ systems. For the v2 release, METplus *use* is supported on systems with a functioning METplus installation, although installation itself is not supported. For more information about METplus, see :numref:`Section %s <MetplusComponent>`.
+
+.. note::
+   If METplus users update their METplus installation, they must update the module load statements in ``ufs-srweather-app/regional_workflow/modulefiles/tasks/<machine>/run_vx.local`` file to correspond to their system's updated installation:
+
+   .. code-block:: console
+      
+      module use -a </path/to/met/modulefiles/>
+      module load met/<version.X.X>
```

Comment:
@jwolff-ncar @michelleharrold Is this what you're looking for?

Reply:
I think that covers what I was trying to get at Thanks!